spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/ApiKeyFetcher.java

```diff
@@ -72,7 +72,7 @@ private Object extractBodyFromResponseEntity(ResponseEntity<?> responseEntity) {
         Optional<String> errorMessage = validateResponseEntityAndCreateErrorMessageInCaseOfProblems(responseEntity);
 
         if (errorMessage.isPresent())
-            throw new ApiKeyFetchingFailedException("A problem occurred referring to the ResponseEntity. "
+            throw new ApiKeyFetchingFailedException("A problem occurred referring to the ResponseEntity received from Admin. "
```

Comment:
```suggestion
            throw new ApiKeyFetchingFailedException("A problem occurred regarding the ResponseEntity received from CoatRack Admin web application. "
```
Or maybe "from the central CoatRack portal/server" see other comment about naming.

Reply:
I tested this now and it seems that the CoatRack admin was not changed to the CoatRack Web Application

![Uploading image.png…]()

Reply:
Then please @ChristophBaierATB make a search also for CoatRack admin and change it to CoatRack Web Application

Reply:
Done.

Reply:
This was solved also on another issue :)